ext/crates/fp/src/limb.rs

```diff
@@ -1,7 +1,6 @@
 use std::ops::Range;
```

Comment:
I think it would be better to make everything in this module `pub(crate)`

Reply:
Everything can be, and should, except for `LimbLength` itself. It's technically in the API, since (e.g.) `BaseVectorP` extends `InternalBaseVectorP` which itself contains a method returning a `LimbLength`.